responsive_html_page.html

```diff
@@ -0,0 +1,381 @@
+    </div>
+
+  </header>
+  <figure class="a">
+    <div class="slide-figure">
+      <div class="img-coverlay">
+        <!---  <div class="img-text"> -->
+        <fieldset class="img-text1">
+          <legend><span>Save the life</span></legend>
+          <p>Lorem ipsum dolor sitAmet, consectetur adipisicing elit. Sed do Eliumd Temper insiden ut labour
+            Et Dolor Magana Alique</p>
+            <button class="button button2 ">Read more</button>
+        </fieldset>
+      </div>
+    </div>
+  </figure>
+  <section class="form-section">
+    <div class="form-field">
+      <form class="form-content">
+        <span>Name</span><br />
+        <input type="text" value="name"><br />
+        <span>Username</span><br />
+        <input type="text" value="username or email"><br />
+        <input type="checkbox"><span class="checkbox">Remember me</span><br />
+        <input type="button" value="SUBMIT">
+      </form>
+      <div class="form-text">
+        <h1>Leave us a message</h1>
+        <hr />
+        <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Lorem ipsum dolor sit amet consectetur
+          adipisicing elit.Ipsa, non, enim numquam atque expedita natus vitae quasi fugit architecto iure
+          ratione debitis pariatur molestiae fugiat? Temporibus molestiae enim atque doloremque....</p>
+        <p>read more >></p>
+      </div>
+    </div>
+  </section>
+  <section>
+    <div class="flex-container">
+
+      <div><i class="fa fa-wrench flex-icons" aria-hidden="true"></i><span>Lnteractive Elements
+          <hr></span>
+        <P>Lorem ipsum dolor sit amet consectetur adipisicing elit. Laudantium natus inventore incidunt delectus
+        </P>
+      </div>
+      <div><i class="fa fa-heartbeat flex-icons" aria-hidden="true"></i><span>Donate
+          <hr></span>
+        <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Laudantium natus inventore incidunt delectus
+        </p>
+      </div>
+      <div><i class="fa fa-usd flex-icons" aria-hidden="true"></i><span>Donate
+          <hr></span>
+        <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Laudantium natus inventore incidunt delectus
+        </p>
+      </div>
+      <div><i class="fa fa-credit-card flex-icons" aria-hidden="true"></i><span>Lnteractive Elements
+          <hr></span>
+        <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Laudantium natus inventore incidunt delectus
+        </p>
+      </div>
+      <div><i class="fa fa-user-plus flex-icons" aria-hidden="true"></i><span>Donate
+          <hr></span>
+        <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Laudantium natus inventore incidunt delectus
+        </p>
+      </div>
+      <div><i class="fa fa-wrench flex-icons" aria-hidden="true"></i><span>Donate
+          <hr></span>
+        <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Laudantium natus inventore incidunt delectus
+        </p>
+      </div>
```

Comment:
Align the html properly. You can use the editor's feature to do alignment on entire file.